test/ctia/http/generative/properties.clj

```diff
@@ -96,16 +130,20 @@
          [NewTool "max-new-tool"]
          [NewVulnerability "max-new-vulnerability"]
          [NewWeakness "max-new-weakness"]
-         ;; TODO enable again once casebook/bundle/data_table
-         ;;does not trigger StackOverFlow Exception
-         ;;[NewCasebook "max-new-casebook"]
-         ]]
+         [NewCasebook "max-new-casebook"]]]
   (fs/->spec (fu/require-all entity)
              kw-ns))
 
 (defn spec-gen [kw-ns]
   (tcg/fmap #(dissoc % :id)
-            (cs/gen (keyword kw-ns "map"))))
+            (cs/gen (keyword kw-ns "map")
+                    (let [;; override data-table Datum, originally `any?` which is
+                          ;; a documented underapproximation and generates huge examples
+                          ;; which don't end up round-tripping via GET anyway.
```

Comment:
it seems that you catched the reason of the StackOverFlow :-)

Reply:
Perhaps, I never saw a StackOverflow myself. I don't have a lot of insights about the problems with the casebook route at the moment unfortunately, other than tiny examples now working.